src/Avalonia.Visuals/Media/GlyphTypeface.cs

```diff
@@ -9,11 +9,9 @@ namespace Avalonia.Media
 {
     public sealed class GlyphTypeface : IDisposable
```

Comment:
Me showing my ignorance now: so what is the difference between a glyph typeface and a simple typeface? Might be worth adding this info to the XML docs.

Reply:
A glyph typeface holds the actual font file and is used by the render. Typeface itself just holds information about FontFamily, Style and Weight. There are situations where you don't want to load the actual font file so these need to be separate.

Reply:
Ah ok, all clear now, pretty much that exact paragraph in a `<remarks>` section for this class would be perfect ;)